### PR TITLE
Use redirect() instead of redirectLink()

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Avoid using deprecated method (since PrestaShop 8).
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes PrestaShop/Prestashop#28147.
| How to test?  | See inside the issue.

### CODE REVIEW

File: controllers/front/validation.php [hunk 1]
```diff
@@ -67,6 +67,6 @@ public function postProcess()
             false,
             $customer->secure_key
         );
-        Tools::redirectLink(__PS_BASE_URI__ . 'order-confirmation.php?key=' . $customer->secure_key . '&id_cart=' . (int) $this->context->cart->id . '&id_module=' . (int) $this->module->id . '&id_order=' . (int) $this->module->currentOrder);
+        Tools::redirect('index.php?controller=order-confirmation?key=' . $customer->secure_key . '&id_cart=' . (int) $this->context->cart->id . '&id_module=' . (int) $this->module->id . '&id_order=' . (int) $this->module->currentOrder);
```
Comment (Progi1984) [inline controllers/front/validation.php:70]:
```suggestion
        Tools::redirect('index.php?controller=order-confirmation&key=' . $customer->secure_key . '&id_cart=' . (int) $this->context->cart->id . '&id_module=' . (int) $this->module->id . '&id_order=' . (int) $this->module->currentOrder);
```

**Review by Progi1984 — COMMENTED** [review]
A Small change and it's ok

**Review by Progi1984 — APPROVED** [review]

**Review by kpodemski — CHANGES_REQUESTED** [review]
This has been fixed already in this PR:
https://github.com/PrestaShop/ps_cashondelivery/pull/32

I suggest waiting for it instead of forcing the author to rebase 👍

**Comment by Progi1984** [comment]
@PrestaEdit Hi, could you resolve conflicts to this PR ? Thnaks

**Comment by PrestaEdit** [comment]
If you say me that it's not for nothing (as Krystian told us: it could be part of another PR), I can.
If not, no :)

**Comment by kpodemski** [comment]
Hello @PrestaEdit 

The PR I mention earlier was merged with a lot of bugfixes and improvements 🎉 

I think the problem from your PR has already been solved. Could you confirm? Thanks!

**Comment by PrestaEdit** [comment]
As it's use redirect() instead of redirectLink(), yes.

We can now close this one.

**Pull request closed without merging**